pkgs/applications/networking/mailreaders/mutt/default.nix

```diff
@@ -27,15 +27,15 @@ with stdenv.lib;
 
 stdenv.mkDerivation rec {
   name = "mutt-${version}";
-  version = "1.11.2";
+  version = "1.11.3";
 
   src = fetchurl {
     url = "http://ftp.mutt.org/pub/mutt/${name}.tar.gz";
-    sha256 = "08w7lbhj5ba2zkjcd0cxkgfiy9y82yhg731xjg9i9292kz1x8p6s";
+    sha256 = "0h8rmcc62n1pagm7mjjccd5fxyhhi4vbvp8m88digkdf5z0g8hm5";
   };
 
   patches = optional smimeSupport (fetchpatch {
-    url = "https://salsa.debian.org/mutt-team/mutt/raw/debian/1.11.2-2/debian/patches/misc/smime.rc.patch";
+    url = "https://salsa.debian.org/mutt-team/mutt/raw/debian/1.11.3-2/debian/patches/misc/smime.rc.patch";
```

Comment:
Neither of these patch links works... So not sure what they are about, except smime then. But yeah, that option is probably broken.

Reply:
I added those on request when fixing #31102. The new link seems to be https://salsa.debian.org/feckert-guest/mutt/tree/master/debian/patches/misc

Reply:
I actually changed the URL in 991d4bf68c7242dfb94f6d276c23749afc8c956d and stated in the commit message that we should use this patch as long as it's applyable.

@ryantm is it possible to avoid breaking such URLs during auto-updates? This got probably broken by f00dcf7f768573bc95e7119c99ff2f9c16fee4d5 and/or 09dae062b6eaefd5691d6eb6b2a096131e5400d0.